decidim-admin/app/mailers/decidim/admin/pseudomize_mailer.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Admin
+    class PseudomizeMailer < Decidim::ApplicationMailer
+      # frozen_string_literal: true
+
+      def notify_admin(user)
```

Comment:
It could be easier to read if we expect `admin` as parameter for `notify_admin`

```suggestion
      def notify_admin(admin)
```